packages/fx-core/src/component/coordinator/index.ts

```diff
@@ -327,14 +330,49 @@ export class Coordinator {
           }
         }
         if (unresolvedPlaceHolders.length === 0) {
```

Comment:
We should continue to call `execute` even if there are unresolvedPlaceholders. Because there could be cross driver references. e.g. Driver2's input depends on Driver1's output. So the placeholder can't be resolve until execution happens.